src/TulsiGenerator/Scripts/bazel_build.py

```diff
@@ -1570,11 +1570,14 @@ def _LinkTulsiWorkspace(self):
     """Links the Bazel Workspace to the Tulsi Workspace ( `tulsi-workspace` )"""
     bazel_workspace = self.bazel_symlink_prefix + os.path.basename(self.workspace_root)
     tulsi_workspace = self.workspace_root + "/tulsi-workspace"
-    if os.path.islink(tulsi_workspace):
-      os.unlink(tulsi_workspace)
-    os.symlink(bazel_workspace, tulsi_workspace)
-    if not os.path.exists(tulsi_workspace):
-      return -1
+    external_local = self.workspace_root + "/external"
```

Comment:
Can you add a comment around this one? I wonder if this something we can remove if the `dSYM` patcher rewrite `external` to `tulsi-workspace/external`?

The entire Xcode project is now relative to the `WORKSPACE` directory.